src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/filters/suppressionsinglefilter/Example1.java

```diff
@@ -1,20 +1,33 @@
 /*xml
 <module name="Checker">
   <module name="SuppressionSingleFilter">
-    <property name="checks" value="JavadocStyleCheck"/>
     <property name="files" value="AbstractComplexityCheck.java"/>
     <property name="lines" value="82,108-122"/>
   </module>
   <module name="SuppressionSingleFilter">
-    <property name="checks" value="MagicNumberCheck"/>
     <property name="files" value="JavadocStyleCheck.java"/>
     <property name="lines" value="221"/>
   </module>
   <module name="SuppressionSingleFilter">
     <property name="message" value="Missing a Javadoc comment"/>
   </module>
+  <module name="TreeWalker">
+    <module name="MagicNumber"/>
+  </module>
 </module>
 */
-
+package com.puppycrawl.tools.checkstyle.filters.suppressionsinglefilter;
 // xdoc section -- start
+public class Example1 {
+  // Checkstyle: Suppress MagicNumberCheck for the MULTIPLIER variable
+  private static final int MULTIPLIER = 5;
```

Comment:
please prove by CLI that this is violation without filter, I doubt.
https://checkstyle.org/cmdline.html#Description

Reply:
please use verifyFilterWithInlineConfigParser method in tests, look around for examples.

Reply:
not done, please do.